stdlib/asm/crypto/stark/random_coin.masm

```diff
@@ -1,106 +1,179 @@
 const.SEED_PTR=4294967293
+const.SEED_PTR_2=4294967294
+const.SEED_PTR_1=4294967295
```

Comment:
nit: I'd probably rename `SEED_PTR` to `SEED_PTR_0` to keep the naming consistent. Also, a short comment on which state part each of these stores would be helpful (e.g., `SEED_PTR_0` stores the capacity word).

Reply:
I have changed the names to more appropriate ones. Let me know if those work.